src/modules/analytics/analytics.service.ts

```diff
@@ -239,15 +250,29 @@ export class AnalyticsService {
         if (measure.toLowerCase().includes('volume')) {
           this.data[series][measure] = this.data[series][measure]
             ? new BigNumber(this.data[series][measure])
-              .plus(eventData[series][measure])
-              .toFixed()
+                .plus(eventData[series][measure])
+                .toFixed()
             : eventData[series][measure];
         }
         if (measure.toLowerCase().includes('volumeUSD')) {
           this.data[series][measure] = this.data[series][measure]
             ? new BigNumber(this.data[series][measure])
-              .plus(eventData[series][measure])
-              .toFixed()
+                .plus(eventData[series][measure])
+                .toFixed()
+            : eventData[series][measure];
+        }
+        if (measure.toLowerCase().includes('floorPrice')) {
+          this.data[series][measure] = this.data[series][measure]
+            ? new BigNumber(this.data[series][measure])
+                .plus(eventData[series][measure])
```

Comment:
don't add price. You should use latest eventData for this.data

Reply:
done